emails/views.py

```diff
@@ -358,6 +383,35 @@ def _sns_message(message_json):
     )
 
 
+def _handle_bounce(message_json):
+    incr_if_enabled('email_bounce', 1)
+    bounce = message_json.get('bounce')
+    bounced_recipients = bounce.get('bouncedRecipients')
+    for recipient in bounced_recipients:
+        try:
+            user = User.objects.get(email=recipient.get('emailAddress'))
+            profile = user.profile_set.first()
+        except User.DoesNotExist:
+            incr_if_enabled('email_bounce_relay_user_gone', 1)
+            # TODO: handle bounce for a user who no longer exists
+            # add to SES account-wide suppression list?
+            return HttpResponse("Address does not exist", status=404)
+        now = datetime.now(timezone.utc)
+        incr_if_enabled(
+            'email_bounce_%s_%s' % (
+                bounce.get('bounceType'), bounce.get('bounceSubType')
+            ),
+            1
+        )
+        if bounce.get('bounceType') == 'Permanent':
+            profile.last_hard_bounce = now
+        if bounce.get('bounceType') == 'Transient':
+            profile.last_soft_bounce = now
```

Comment:
```suggestion
        if bounce.get('bounceType') == 'Permanent':
            profile.last_hard_bounce = now
            incr_if_enabled('email_hard_bounce', 1)
        if bounce.get('bounceType') == 'Transient':
            profile.last_soft_bounce = now
            incr_if_enabled('email_soft_bounce', 1)
```
More specific metrics?